ethereum_history_api/oracles/src/noir/noir_js/encode.ts

```diff
@@ -1,13 +1,15 @@
 import { isHex } from 'viem';
 
+const BYTE_HEX_LENGTH = 2;
```

Comment:
`NIBBLE_IN_BYTE` (?)
`BYTE_NIBBLE_LENGTH` (?)

Reply:
```
const hexValue = value.toString(16).padStart(length * BYTE_HEX_LENGTH, '0');
const hexValue = value.toString(16).padStart(length * BYTE_NIBBLE_LENGTH, '0');
```

Which line is more readable?
I think that in this context - nibbles are irrelevant. It's just relevant that each byte takes 2 so we pad to len * byte_len to get a correct value.